### ISSUE TITLE
[v3] No supporting normalizer found

### ISSUE THREAD

**Issue by tristanbes** [algolia/search-bundle#148]
After applying the #147 patch, it will give:

```
In Serializer.php line 164:
Could not normalize object of type AppBundle\Entity\Cv, no supporting normalizer found.
```

Yet, my `CvNormalizer`
```php
use AppBundle\Entity\Cv;

 class CvNormalizer implements NormalizerInterface, SerializerAwareInterface
{
    use SerializerAwareTrait;

     ....

    public function supportsNormalization($data, $format = null)
    {
        return $data instanceof Cv;
    }
}
```

```xml
<service id="cv_normalizer" class="AppBundle\Serializer\Normalizer\CvNormalizer" public="false">
    <tag name="serializer.normalizer" />
</service>
```

**Comment by tristanbes** [comment]
found the bug.

**Comment by julienbourdeau** [comment]
Fixed by #153
